apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -531,59 +451,86 @@ class TopInstructions extends Component {
     () => {
       const latestFeedback = this.state.feedbacks[0];
       if (!this.state.teacherViewingStudentWork && latestFeedback) {
-        $.ajax({
-          url: `/api/v1/teacher_feedbacks/${
-            latestFeedback.id
-          }/increment_visit_count`,
-          method: 'POST',
-          contentType: 'application/json;charset=UTF-8',
-          headers: {'X-CSRF-Token': this.state.token}
-        });
+        incrementVisitCount(latestFeedback.id, this.state.token);
       }
     },
     5000,
     {leading: true}
   );
 
+  isViewingAsStudent = () => {
+    return this.props.viewAs === ViewType.Student;
+  };
+
+  isViewingAsTeacher = () => {
+    return this.props.viewAs === ViewType.Teacher;
+  };
+
+  setInstructionsRef(ref) {
+    if (ref) {
+      this.instructions = ref;
```

Comment:
can you declare this by setting it to null right under the constructor? i'm pretty opposed to setting properties that haven't been declared anywhere.